app/controllers/use_case_controller.rb

```diff
@@ -3,6 +3,7 @@
 # * permitted_params
 class UseCaseController < ApplicationController
   before_filter :authenticate_user!
+  before_filter ->{authorize self.model_class.new}, only: [:new, :create]
```

Comment:
I don't think you need to authorize `self.model_class.new` separately due to `authorizing_object`